### DISCUSSION TITLE
大家好

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#1620] [其他]
我是来自黑曜石军营中奥波勒斯小队的11号，真名你无需知道，而且我已经摒弃真名，毕竟真名会令人感到‘软弱’，我诚邀各位加入保卫新艾丽都的部队中（而且服兵役是每个新艾丽都成年物种的义务）
具体报名方式请点击此链接   #1605（对于本文某些乱码我十分抱歉，那是因为部队中电脑的字符编码与各位的不同，后面没有乱码是我后面用手机编辑的）

**Comment by gaojiaobei** [comment]
害怕😰，在绳网征兵真的不是把绳匠骗进去杀吗
听行内人士透露会动用私刑的啊，好可疑啊🤨

**Comment by Soldier11-ObsidianBarracks** [comment]
诽谤啊，这是纯纯的诽谤！
我们征兵，直接加入新兵营，绳匠我们会根据他们的能力分发到不同的部门，绝对没有私刑（确信

**Comment by AnbulesPeo** [comment]
有传言称您与知名绳匠“法厄同”在私下有不合规程的合作，请问您对于此事是如何看待的呢

**Comment by Soldier11-ObsidianBarracks** [comment]
法厄同？你说那个传奇绳匠？我不清楚，你该不会要说random play的店长是法厄同吧，笑死，她也是这样说的，怎么可能啊？店长要真是法厄同还会与我一起抓叛军？
